Microsoft.VisualStudio.DSC/Microsoft.VisualStudio.DSC.psm1

```diff
@@ -55,8 +55,13 @@ class VSComponents
         $this.Get()
         $requestedComponents = $this.components
 
-        if($this.vsConfigFile)
+        if ($this.vsConfigFile)
```

Comment:
nit: looks like the style in this PS script is no space in `if(`

Reply:
I think it's a mix, there are some `if (` as well haha but I can make it consistent in next bug fix